doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-security-management-con.adoc

```diff
@@ -0,0 +1,7 @@
+[id='business-central-security-management-con_{context}']
+
+= Security management
+
+Security management describes how you can manage the users, groups, and permissions with the help of user interface in order to configure who can access the different available resources and features.
```

Comment:
Security management is the process of managing users, groups, and permissions.  
Delete this
 with the help of user interface in order to configure who can access the different available resources and features.

Rethink this paragraph. There is repetition. Try consolidate it.

Reply:
I have removed the repetitive content.